Open.Nat/Utils/Extensions.cs

```diff
@@ -101,6 +101,24 @@ internal static string ToPrintableXml(this XmlDocument document)
             }
         }
 
+#if NET35
+        public static Task<TResult> TimeoutAfter<TResult>(this Task<TResult> task, TimeSpan timeout)
```

Comment:
Currently this method always throws a TimeoutException when compiled in release mode (trying to use this with Unity). Is this pull request still alive?